app/controllers/merchants/orders_controller.rb

```diff
@@ -1,7 +1,12 @@
 class Merchants::OrdersController < Merchants::BaseController
   def show
     @order = Order.find(params[:id])
-    @merchant_items = @order.items.where(user: current_user)
-    @merchant_order_items = @order.order_items.where(item_id: @merchant_items)
+    items = @order.items.where(user: current_user)
+
+    @merchant_items = {}
```

Comment:
Could also do with inject, but this definitely works as well.

Reply:
Agreed. But then I would have to use inject.  🙉